### ISSUE TITLE
Use legislation.gov.uk for all references

### ISSUE THREAD

**Issue by nikhilwoodruff** [PolicyEngine/policyengine-uk#20]
Sources are one area of the documentation which are pretty inconsistent here - we should aim to make them known wherever they have been used. At the moment, the vast majority of implementations have used either GOV.UK or the 2016-2020 UKMOD country report.

**Comment by nikhilwoodruff** [comment]
I don't think we should commit to this, given e.g. OBR parameters are from the OBR.

**Comment by MaxGhenis** [comment]
Yes, but we should avoid citing the UKMOD country report. If we've moved to primary sources already feel free to re-close.
